<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinalizeTransaction</class>
 <widget class="QWidget" name="FinalizeTransaction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>713</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>391</width>
    <height>713</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>731</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>371</width>
        <height>693</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,1,1,1,1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="horizontalFrame">
         <property name="styleSheet">
          <string notr="true">background-color: red;</string>
         </property>
         <layout class="QHBoxLayout" name="warningLayout" stretch="0,9">
          <item>
           <widget class="QPushButton" name="warningIcon">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: transparent;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons_black/alert-circle.png</normaloff>:/icons_black/alert-circle.png</iconset>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="warningLabel">
            <property name="text">
             <string>Płatność tylko gotówką</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>203</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>203</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QLabel" name="buyersLabel">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>391</width>
            <height>16</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Dane odbiorcy przesyłki</string>
          </property>
         </widget>
         <widget class="QLabel" name="nameAndSurnameLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>301</width>
            <height>16</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Imie i nazwisko</string>
          </property>
         </widget>
         <widget class="Line" name="line">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>30</y>
            <width>711</width>
            <height>16</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>311</width>
            <height>31</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="zipCodeAndCityLayout">
           <item>
            <widget class="QLabel" name="zipCodeLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Kod pocztowy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="cityLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Miasto</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="addressLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>311</width>
            <height>16</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Adres</string>
          </property>
         </widget>
         <widget class="QLabel" name="emailLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>140</y>
            <width>301</width>
            <height>16</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>email</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ProduktyLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Produkty</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="productsLayout"/>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <widget class="QLabel" name="deliveryOptionsLabel">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>10</y>
            <width>191</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Opcje dostawy</string>
          </property>
         </widget>
         <widget class="Line" name="line_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>30</y>
            <width>721</width>
            <height>31</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
         <widget class="QFrame" name="optionsFrame">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>178</width>
            <height>92</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="lhdRadioButton">
             <property name="text">
              <string>Dostawa LHD</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">deliverGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="spuRadioButton">
             <property name="text">
              <string>Dostawa SPU</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">deliverGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="personalCollectionRadioButton">
             <property name="text">
              <string>Odbiór osobisty na UKSW</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">deliverGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,1">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="priceStringLabel">
            <property name="text">
             <string>Cena:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="priceDoubleLabel">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>9999.99 zł</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="confirmButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>75</height>
             </size>
            </property>
            <property name="text">
             <string>Potwierdzam
zamówienie</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="deliverGroup"/>
 </buttongroups>
</ui>
